src/test/resources/two.properties

```diff
@@ -1,30 +1,24 @@
-plugin.zipMapper.class = org.icatproject.ids.storage_test.ZipMapper
-
-plugin.main.class = org.icatproject.ids.storage_test.MainFileStorage
-plugin.main.dir = ${HOME}/data/ids/main/
-
-cache.dir = ${HOME}/data/ids/cache/
-preparedCount = 100
-processQueueIntervalSeconds = 2
+plugin.zipMapper.class=org.icatproject.ids.storage_test.ZipMapper
+plugin.main.class=org.icatproject.ids.storage_test.MainFileStorage
+plugin.main.dir=${HOME}/data/ids/main/
+cache.dir=${HOME}/data/ids/cache/
+preparedCount=100
+processQueueIntervalSeconds=2
 rootUserNames db/root
```

Comment:
Same as above.

But suggest to fix the missing `=` between `rootUserNames` and `db/root`.